### ISSUE TITLE
adds prefixes on -js-display: flex;

### ISSUE THREAD

**Issue by m-e-h** [postcss/autoprefixer#608]
When using [Flexibility](https://github.com/10up/flexibility), autoprefixer treats `-js-display: flex` the same as `display: flex`
```
    -js-display: -webkit-flex;

    -js-display: -ms-flexbox;

    -js-display: flex;

    display: -webkit-flex;

    display: -ms-flexbox;

    display: flex;
```

**Comment by ai** [comment]
https://github.com/7rulnik/postcss-flexibility

**Comment by m-e-h** [comment]
yes I've tried that. When I use it after autoprexer I get 
```
    -js-display: flex;

    display: -webkit-flex;

    -js-display: flex;

    display: -ms-flexbox;

    -js-display: flex;

    display: flex;
```
I'll leave an issue over there.

**Comment by ai** [comment]
Hm. I will fix it tomorrow.

**Comment by m-e-h** [comment]
Thanks!

**Comment by ai** [comment]
Done 518e982

**Comment by ai** [comment]
Released in 6.3.1

**Comment by m-e-h** [comment]
Awesome! Thanks!
